.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Release workflow
+jobs:
+  build-phar:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: 7.4
+          ini-values: memory_limit=2G, display_errors=On, error_reporting=-1, phar.readonly=0
+          tools: pecl, phive
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: composer-
+
+      - name: Install Composer dependencies
+        run: |
+          composer install --no-progress --prefer-dist --optimize-autoloader
+
+      - name: Install phive tools
+        run: phive install --trust-gpg-keys 2A8299CE842DD38C
+
+      - name: Create build dir
+        run: make phar
+
+      - name: Upload PHAR file
+        uses: actions/upload-artifact@v2.2.0
+        with:
+          name: prophecy-phpunit.phar
+          path: build/prophecy-phpunit.phar
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
```

Comment:
what if we tag a prerelease tag instead of a stable one ?

Reply:
I think we need to discuss how to handle those, I didn't find any pre-releases in this package. 
Rethinking this, we need to release every time prophecy itself is released.
This would mean that we need to connect the projects more tightly. Of course this can be automated some in some way. But before we spend to much time on this I would like to know your opinion about is.

Reply:
if the releasing of the phar needs to depend on releasing both prophecy and prophecy-phpunit, it should not be managed in the releases of the prophecy-phpunit repository itself, as that would mess things up. We should either have separate phars for each package or a separate repo managing the combined phar

Reply:
For example, when you would release a bugfix for `prophecy` people using `prophecy-phpunit` would need to get that patch. Seen the amount of code in this repo I don't expect too many changes. 

But you might be right. We could setup a fully automated repo which is watching both repo's for changes and creates a new phar when needed.

Since this is kind of an edge case I don't want to take too much of your time. So maybe it is better to create a separate project first. And move it under the phpspec flag later if we are going to share maintenance.
That was also the mean reason for me to open an issue first before creating this PR.